src/js/vendor/buffer/utf8ToBytes.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable */
+
+/**
+ * Convert a string to a byte array
+ *
+ * This is a utf8ToBytes function from the buffer module (with added types)
+ * https://github.com/feross/buffer/blob/795bbb5bda1b39f1370ebd784bea6107b087e3a7/index.js#L1956
+ *
+ * License: MIT (https://github.com/feross/buffer)
+ */
```

Comment:
@marandaneto Is the docs for the external lib okay like this?

Reply:
I believe so, please double check either with @bruno-garcia or the open source team, they have more experience with vendoring and licenses.

Reply:
Important to check the license: 
https://github.com/feross/buffer/blob/795bbb5bda1b39f1370ebd784bea6107b087e3a7/LICENSE

with MIT we're good. But you need to add the license to the header of the file. Or the whole file to a folder for example

Reply:
Okay, cool. Will do.